update content and add deployment github action

suejinkim20 opened the pull request.

mbwatson reviewed (changes requested):
:+1: looks good!

please clean up those eslint warnings :grin:

mbwatson commented on src/components/dates-table.js:
```diff
@@ -13,7 +13,7 @@ export const DatesTable = ({content, type}) => {
 
   const datesArray = createDatesArray(content)
-
+console.log(datesArray)
```
can probably lose this log

mbwatson reviewed (changes requested):
i see a few eslint warnings
- Warning: React does not recognize the `backgroundColor` prop on a DOM element
- Warning: Each child in a list should have a unique "key" prop
- warning  'rows' is assigned a value but never used  no-unused-vars

suejinkim20 commented:
> i see a few eslint warnings
> 
> * Warning: React does not recognize the `backgroundColor` prop on a DOM element
> * Warning: Each child in a list should have a unique "key" prop
> * warning  'rows' is assigned a value but never used  no-unused-vars

hi @mbwatson ! 

i've removed the two console.logs and addressed the remaining browser warnings.

there are two remaining eslint warnings that are both related to future utilization of the Monday plugin for Dates. these errors should disappear when i implement the logic to pull in the dates from the Monday board rather than the current hardcoded method. the two errors are as follows:

```
/star-website/src/components/sections/important-dates.js
 7:9  warning  'rows' is assigned a value but never used  no-unused-vars
```
  
```
 /star-website/src/components/sections/programs/starShipPanel.js
 30:9  warning  'data' is assigned a value but never used  no-unused-vars
```

please let me know if there are any other errors i missed. thank you!

mbwatson reviewed (approved):
:+1:
